lib/xcodeproj/constants.rb

```diff
@@ -138,13 +138,13 @@ module Constants
         'MTL_ENABLE_DEBUG_INFO'             => 'NO',
       }.freeze,
       [:ios] => {
-        'SDKROOT'                           => 'iphoneos',
+        'SDKROOT'                           => `xcrun xcode-select --print-path`.strip + "/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS" + LAST_KNOWN_IOS_SDK + ".sdk",
```

Comment:
I get it. Maybe change the SDKROOT is not a good idea. I' ll close this pull request.